plugins/woocommerce/includes/class-wc-order.php

```diff
@@ -2286,4 +2286,13 @@ public function get_order_item_totals( $tax_display = '' ) {
 	public function is_created_via( $modus ) {
 		return apply_filters( 'woocommerce_order_is_created_via', $modus === $this->get_created_via(), $this, $modus );
 	}
+
+	/**
+	 * Attempts to restore the specified order back to its original status (after having been trashed).
+	 *
+	 * @return bool If the operation was successful.
+	 */
+	public function untrash(): bool {
+		return (bool) $this->data_store->untrash_order( $this );
```

Comment:
perhaps we should do a method_exist check here and fallback to wp_untrash_post for custom implementation of a data store which does not have yet this method. otherwise this will cause fatal. wdyt?

Reply:
Ah, you are right, I forgot that this goes via a `__call` magic method.